tests/autoyast/installation.pm

```diff
@@ -105,6 +105,13 @@ sub run {
     my $timer      = 0;    # Prevent endless loop
 
     mouse_hide(1);
+
+    # QAM CaaSP specific
+    if (is_caasp && check_var('FLAVOR', 'CaaSP-DVD-Incidents')) {
+        assert_screen('bios-boot', 1200);
```

Comment:
I guess it will be easier to adjust needles array for this scenario, and just put one needle there and using same code as before. I also not sure if it's really what you want to see, as you can ignore warning/errors in the profile. This loop helps to fail test before it times out in 2 hours, which is good from many perspectives. So I would leave at least 'bios-boot' and 'linuxrc-install-fail'. You may also introduce needle which is there when system is already booted, then we don't care that we have missed bios-boot

Reply:
yeah, doesn't make much sense. Can you reject this PR?